test/support/adapter_assertions.ex

```diff
@@ -5,6 +5,24 @@ defmodule Cog.AdapterAssertions do
     assert response == expected_message
   end
 
+  # Because the raw template returns encoded json joined by newlines we have to
+  # add in newlines and then pull out the single result or return a list of
+  # result for those cases.
+  def assert_payload(%{"response" => response}, expected_payload) do
+    response = response
+    |> String.replace(~r/^}/m, "},")
+    |> String.rstrip(?,)
+
+    payload = case Poison.decode!("[#{response}]", keys: :atoms) do
```

Comment:
This is pretty gnarly. The comment above the function helps a little bit.

We don't know how many json results are printed (separated by newlines) so to make sure we can always parse it, I need to add commas and wrap it in an array before parsing. Eventually I hope these tests just intercept things on the message queue, but for now to get these tests passing, this was the best solution.